x/stake/types/stake_migration.go

```diff
@@ -0,0 +1,79 @@
+package types
+
+import (
+	"fmt"
+	"math/big"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+const (
+	StakeMigrationChannel = "stakeMigration"
+
+	StakeMigrationChannelID sdk.ChannelID = 17
+
+	TagStakeMigrationSendSequence = "StakeMigrationSendSequence"
+
+	MsgTypeSideChainStakeMigration = "stake_migration"
+
+	StakeMigrationRelayFee int64 = 1000000 // decimal 8
+)
+
+type StakeMigrationSynPackage struct {
+	OperatorAddress  sdk.SmartChainAddress
+	DelegatorAddress sdk.SmartChainAddress
+	RefundAddress    sdk.AccAddress
+	Amount           *big.Int
+}
+
+type MsgSideChainStakeMigration struct {
+	Validator        sdk.ValAddress        `json:"validator"`
```

Comment:
what about `ValidatorSrcAddr` and `ValidatorDstAddr` just like the `MsgSideChainRedelegate` message?

Reply:
Done